tests/e2e/fixtures/ncc/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@prisma/client": "__dynamic__",
-    "@vercel/ncc": "0.29.2",
+    "@vercel/ncc": "0.33.4",
```

Comment:
Why is this dep being updated in the same PR as Prisma deps?

Reply:
I thought it was a reason for this issue https://github.com/AhmedElywa/nexus-prisma/runs/6112252422?check_suite_focus=true#step:8:558

<img width="1149" alt="image" src="https://user-images.githubusercontent.com/4535142/164529985-b572fa67-eea3-4041-8690-3d69a314c9e9.png">

Reply:
This can be updated in a new PR.

Reply:
this change can be undo we don't need that